src/build/content/prerendered.ts

```diff
@@ -169,6 +170,11 @@ export const copyPrerenderedContent = async (ctx: PluginContext): Promise<void>
                   throw new Error(`Unrecognized content: ${route}`)
               }
 
+              // Netlify Forms are not support and require a workaround
+              if (value.kind === 'PAGE') {
```

Comment:
does it also need to check `APP_PAGE`? (next@canary use that kind for app router page cache)

Reply:
Ah, good catch 🙏 fixed in https://github.com/netlify/next-runtime/pull/2512/commits/faed5fafe6b564fdcf97300e97e971b4d1a59bad